dash_bio/component_factory/_clustergram.py

```diff
@@ -67,66 +66,65 @@ def Clustergram(
-    distance from the observations (see
-    scipy.spatial.distance.pdist). (Default: scs.distance.pdist)
-- link_fun (function; optional): Function to compute the linkage
-    matrix from the pairwise distances (see
-    scipy.cluster.hierarchy.linkage). (Default: sch.linkage)
-- color_threshold (dict; optional): Maximum linkage value for which
-    unique colors are assigned to clusters; 'row' for rows, and 'col'
-    for columns. (Default: {'row': 0, 'col': 0})
-- optimal_leaf_order (bool; optional): Enabling/disabling of the
+    row-clustered data.
+- row_dist (string; default 'euclidean'): String specifying the
+    distance metric for rows. It will be passed as the argument
+    'metric' into the function specified in dist_fun (see
+    scipy.spatial.distance.pdist).
+- col_dist (string; default 'euclidean'): String specifying the
+    distance metric for columns. It will be passed as the argument
+    'metric' into the function specified in dist_fun (see
+    scipy.spatial.distance.pdist).
+- dist_fun (function; default scipy.spatial.distance.pdist): Function
+    to compute the pairwise distance from the observations (see
+    scipy.spatial.distance.pdist).
+- link_fun (function; default scipy.cluster.hierarchy.linkage): Function to
+    compute the linkage matrix from the pairwise distances (see
+    scipy.cluster.hierarchy.linkage).
+- color_threshold (dict; default {'row': 0, 'col': 0}): Maximum
+    linkage value for which unique colors are assigned to clusters;
+    'row' for rows, and 'col' for columns.
+- optimal_leaf_order (bool; default False): Enabling/disabling of the
     option to determine leaf order that maximizes similarity between
-    neighboring leaves. (Default: False)
-- color_map (list; optional): The colorscale for the heatmap. Each
-    list element contains two elements; the first element refers to
-    the portion of the maximum data point under which a cell will be
-    colored, and the second element refers to the color. e.g., a
-    colorscale [[0.0, 'white'], [0.5, 'gray'], [1.0, 'black']] mean
-    that for all cells with a value in the 50th or lower percentile of
-    the dataset, the color on the heatmap would be white; all cells
-    with a value in the 50th or higher percentile, excluding the 100th
-    percentile, would be gray; and the cell(s) in the 100th percentile
-    would be colored black. (Default: [[0.0, 'rgb(255,0,0)'], [0.5,
-    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']])
+    neighboring leaves.
+- color_map (list; default [[0.0, 'rgb(255,0,0)'], [0.5,
+    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']]): The colorscale for the
+    heatmap. Each list element contains two elements; the first
+    element refers to the portion of the maximum data point under
+    which a cell will be colored, and the second element refers to the
+    color, e.g., a colorscale [[0.0, 'white'], [0.5, 'gray'], [1.0,
+    'black']] means that for all cells with a value below the 50th
```

Comment:
```suggestion
    'black']] means that for all cells with a value in the 49th
```

Reply:
I'm a bit confused here -- you used "at" and not "in" in a previous suggestion (https://github.com/plotly/dash-bio/pull/379#discussion_r304038202). 

You also mentioned not using the word "value" here: https://github.com/plotly/dash-bio/pull/379#discussion_r298771236

In the interest of consistency, we should make sure everything has the same wording!

Reply:
> I'm a bit confused here -- you used "at" and not "in" in a previous suggestion ([#379 (comment)](https://github.com/plotly/dash-bio/pull/379#discussion_r304038202)).

They mean different things: "in the 49th percentile" means "at the 1st, 2nd, 3rd, ..., 48th, or 49th percentile."

> You also mentioned not using the word "value" here: [#379 (comment)](https://github.com/plotly/dash-bio/pull/379#discussion_r298771236)

Sorry, my wording suggestions at https://github.com/plotly/dash-bio/pull/379#discussion_r304037899 were about the English, the syntax, not the math, the stats anymore.

> In the interest of consistency, we should make sure everything has the same wording!

Absolutely: within each paragraph (those than explain the white/grey/black colours) and between paragraphs (it looks like there are 3 instances of these descriptions).

Reply:
@mkcor Might be faster for you to do it, if that's OK with you!

Reply:
> @mkcor Might be faster for you to do it, if that's OK with you!

@shammamah all good, on it!